README.md

```diff
@@ -12,108 +59,157 @@ A REST API to support Reef Guide (AIMS), built with Express, TypeScript, Zod and
 - Environment-based configuration with Zod validation
 - Basic docker compose
 
-## Prerequisites
+## Quick Start
 
-- Node.js (v18+)
-- AWS CLI configured with appropriate permissions
-- Docker (for local development with Prisma)
+Get up and running with the API in minutes:
 
-## Setup (local)
+```bash
+# Clone the repository
+git clone <https://github.com/open-AIMS/reefguide-web-api.git>
+cd <reefguide-web-api>
```

Comment:
why the carrots? folder name will be `reefguide-web-api` based on previous command